card/src/main/java/com/airwallex/android/card/CardComponent.kt

```diff
@@ -25,6 +27,10 @@ class CardComponent : ActionComponent {
     private var dccCallback: DccCallback? = null
     private var listener: Airwallex.PaymentResultListener? = null
 
+    override fun init(application: Application) {
```

Comment:
Can we call it `initialize` for consistency? Init feels like it's system `init {}`

Reply:
ok